src/codecs/webp/extended.rs

```diff
@@ -147,6 +147,54 @@ impl ExtendedImage {
         Frames::new(Box::new(frame_iter))
     }
 
+    pub(crate) fn as_frames<'a>(&'a self) -> Frames<'a> {
```

Comment:
Thanks for adding the `as_frames` API. That's very useful. However, does the code have to be duplicated with `into_frames`, or could `into_frames` cust call `as_frames`?

Reply:
@fintelia I think having `as_frames` for #1869 would be good. Should it maybe take a `&mut self` just to be safe?